packages/tests/tsconfig.dev.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig",
   "compilerOptions": {
-    "module": "esnext",
+    "module": "es2020",
```

Comment:
we actually need `esnext` for things to work in ESM. What issue did you encounter? or is that the NodeJS 18 issue?

Reply:
Note that if we can use `es2020` here and it still works, that's great. It also mean that we can remove this line as `tsconfig.json` already sets `module` to `es2020`